lightning-tag/basic-implementation.md

```diff
@@ -0,0 +1,56 @@
+# Basic Lightning Tag Implementation 
+
```

Comment:
@ErikMHummel  - where is this file going to live in the structure of the site?

Reply:
We would need to update `_sidebar.md` to add it and then a snippet on 'index.md' to call it out.

Reply:
@ErikMHummel - can you work with Charlie to show him how to do that?

Reply:
Yup, I have that on my list for today.

Reply:
Done @noahpc, please review and if all looks good merge.